src/model/migrations.ts

```diff
@@ -66,6 +67,13 @@ export default async function runMigrations() {
   const currentVersion = Number(await getMigrationVersion());
   const migrations = [];
   const mmkv = new MMKV();
+  const storageAR = new MMKV({
```

Comment:
Just personal preference, but I'd keep full names like `storageAspectRatio` etc.

I saw the bottom part first and was confused by `storageAR` name.

Reply:
tweaked